### PR TITLE
Feat/change area manager to area owner

### PR DESCRIPTION
**Pull request by sergiomarrocoli**
area_manager column renamed area_manager to better match question wording

Testing
- run migrations
- start or edit a commitment and save it with tab 2 question 4 filled in
- open the dit form again and check the value appears in the question field

### CODE REVIEW

File: test/controllers/links_controller_test.rb [hunk 1]
```diff
@@ -5,14 +5,12 @@ class LinksControllerTest < ActionDispatch::IntegrationTest
```
Comment (yndajas) [inline test/controllers/links_controller_test.rb:5]: Getting an error on this test. Not sure how it related to this PR anyway though?

Screenshot 2022-03-28 at 17 45 14

```
Error:
LinksControllerTest#test_should_update_a_link:
NoMethodError: undefined method `name' for #<Link:0x000000011ebf2e70>
    test/controllers/links_controller_test.rb:10:in `block in <class:LinksControllerTest>'
```

File: test/models/criterium_test.rb [hunk 2]
```diff
@@ -10,10 +10,9 @@ class CriteriumTest < ActiveSupport::TestCase
   test "should not save with both 'none of the above' other options" do
```
Comment (yndajas) [inline test/models/criterium_test.rb:10]: And errors here:

Screenshot 2022-03-28 at 17 47 32

```
.F

Failure:
CriteriumTest#test_should_not_save_with_both_'none_of_the_above'_other_options [/Users/yndajas/dev/UNEP-WCMC/action-agenda/test/models/criterium_test.rb:13]:
Expected false to be truthy.


bin/rails test test/models/criterium_test.rb:10

F

Failure:
CriteriumTest#test_should_be_invalid_for_commitment_if_it_has_negative_values [/Users/yndajas/dev/UNEP-WCMC/action-agenda/test/models/criterium_test.rb:25]:
Expected true to be nil or false


bin/rails test test/models/criterium_test.rb:20

F

Failure:
CriteriumTest#test_should_give_the_correct_failure_messages [/Users/yndajas/dev/UNEP-WCMC/action-agenda/test/models/criterium_test.rb:34]:
--- expected
+++ actual
@@ -1 +1 @@
-[]
+["Your commitment is being made by a government"]
```

**Review by yndajas — CHANGES_REQUESTED** [review]
Form seems to work, but getting a few errors in the tests that have been updated in this PR

Also, I noticed the wording of the question suggests this is the area owner and their role - is it worth naming the column etc to represent the broader scope of the data (e.g. `area_owner_and_role`), or do you think `area_owner` is cleaner?

Screenshot 2022-03-28 at 17 50 31

File: test/models/criterium_test.rb (hunk 2 above)
Comment (sergiomarrocoli) [reply test/models/criterium_test.rb:10]: not getting errors on this branch now, but happening on other branches. maybe needs db reset?

Changed column name as requested

**Review by yndajas — APPROVED** [review]
Looks good, but conflicts to be resolved before merging

**Pull request merged**